pkg/apiserver/vshn/redis/get.go

```diff
@@ -0,0 +1,54 @@
+package redis
+
+import (
+	"context"
+	"fmt"
+
+	appcatv1 "github.com/vshn/appcat/apis/appcat/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/registry/rest"
+)
+
+var _ rest.Getter = &vshnRedisBackupStorage{}
+
+func (v *vshnRedisBackupStorage) Get(ctx context.Context, name string, opts *metav1.GetOptions) (runtime.Object, error) {
+
+	namespace, ok := request.NamespaceFrom(ctx)
+	if !ok {
+		return nil, fmt.Errorf("cannot get namespace from context")
+	}
+
+	instances, err := v.vshnRedis.ListVSHNRedis(ctx, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	redisSnap := &appcatv1.VSHNRedisBackup{}
+
+	for _, instance := range instances.Items {
+		ins := instance.Status.InstanceNamespace
+		snap, err := v.snapshothandler.Get(ctx, name, ins)
+		if err != nil {
+			if apierrors.IsNotFound(err) {
+				continue
+			}
+			return nil, err
+		}
+
+		redisSnap = &appcatv1.VSHNRedisBackup{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: trimStringLenght(*snap.Spec.ID),
+			},
+			Status: appcatv1.VSHNRedisBackupStatus{
+				ID:       *snap.Spec.ID,
+				Date:     *snap.Spec.Date,
```

Comment:
Bit of a nitpick, but given that these two fields are pointers, they might be empty. And de-referencing them can lead to a panic. I'd prefer to be defensive and check for `nil`.